src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -495,6 +532,7 @@ public static String determineFileType(File f, String fileName) throws IOExcepti
                 logger.fine("mime type recognized by extension: "+fileType);
             }
         } else {
+            //ToDo - if the extension is null, how can this call do anything
```

Comment:
```suggestion
```

I'm not sure why this todo is here so I'm suggesting we delete it. In this PR...

- https://github.com/IQSS/dataverse/pull/8744

... we added support for detecting files like Makefile or Dockerfile. Having no extension is legit. Maybe I'm just misunderstanding the todo. 🤷